x-pack/plugins/encrypted_saved_objects/server/plugin.mock.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin } from './plugin';
+type EncryptedSavedObjectsPlugin = ReturnType<Plugin['setup']>;
+
+type Schema = PublicMethodsOf<EncryptedSavedObjectsPlugin>;
```

Comment:
__nit:__ `Schema` ---> `SetupContract` or something like this to be consistent with the naming we have in core mocks?

Reply:
Makes sense, made the change to `Contract` as discussed https://github.com/elastic/kibana/pull/37043/commits/f3d4dc4fe6ba85630ac50456838d656c1097506e.